### ISSUE TITLE
App immediately crashes post update 0.5.1 (New 3DS)

### ISSUE THREAD

**Issue by DragonWario** [windows-server-2003/ThirdTube#138]
Basically what the title says. As soon as I booted after the update it crashed. Downgrading to 0.5.0 again worked fine though. 

Hardware: New 3DS XL
Luma build: Luma3DS v11.0-6b633bc4 (https://github.com/Core-2-Extreme/Luma3DS/releases/)
System Firmware: 11.16.0-49U
Installed via .cia

Github won't accept the .dmp dump file but I have it as well if you need it.

**Comment by cooolgamer** [comment]
Yeah I have the same issue, some videos crashes the system even if they're not played

**Comment by brainwo** [comment]
Same issue here, it says translation error

**Comment by brainwo** [comment]
I found the issue, it's in https://github.com/windows-server-2003/ThirdTube/commit/74aa65d0ca5321ba6393b18981b777092df26dfe. I think some 3DS just couldn't render the font, resulted in translation error which should be solved by changing Luma region on the title, but I've tried to change it to EN-US and still have the same error.

I have my own fork that reverts that commit with some UI tweaks included. It's working and I use it myself.
https://github.com/brainwo/thirdtube

**Comment by cooolgamer** [comment]
thanks @brainwo it worked! ^^

**Comment by brainwo** [comment]
Made a release on https://github.com/brainwo/thirdtube/releases/tag/v0.5.1-fixed

**Comment by windows-server-2003** [comment]
I would like to know if this is fixed in v0.5.2

**Comment by cooolgamer** [comment]
it is, the issue can be closed
